server/rpm/Makefile

```diff
@@ -6,35 +6,36 @@
 
 CWD = $(shell pwd -L)
 
-SERVER = $(dir ${CWD})
-PBENCHTOP = $(shell dirname ${SERVER})
-TMPDIR = /tmp/opt
+component = server
 
-VERSION = $(shell cat ${SERVER}/VERSION)
+PBENCHTOP := $(shell git rev-parse --show-toplevel)
+VERSION := $(shell cat ${PBENCHTOP}/${component}/VERSION)
+MAJORMINOR := $(shell grep -oE '[0-9]+\.[0-9]+' ${PBENCHTOP}/${component}/VERSION)
+
+prog = pbench-server
+arch = noarch
 
 DISTRO = $(shell uname -r | python -c 'import sys; print(sys.stdin.read().split(".")[-2])')
 
 PYTHON = python3
 
-RPMSRC = ${HOME}/rpmbuild/SOURCES
-RPMSRPM = ${HOME}/rpmbuild/SRPMS
-RPMSPEC = ${HOME}/rpmbuild/SPECS
-
-prog = pbench-server
-arch = noarch
-
+# temp directory for packing up the tarball
+TMPDIR = /tmp/opt
 TBDIR = ${TMPDIR}/${prog}-${VERSION}
 
 USE_GIT_SHA1 = yes
-sha1 = $(shell git rev-parse --short HEAD)
+sha1 := $(shell git rev-parse --short HEAD)
 seqno := $(shell if [ -e ./seqno ] ;then cat ./seqno ;else echo "1" ;fi)
 
-rpm:  srpm
-	rpmbuild -bb ${HOME}/rpmbuild/SPECS/${prog}.spec
+# By default we only build a source RPM
+all: srpm
+
+rpm: srpm
+	rpmbuild -bb ${RPMSPEC}/${prog}.spec
 
-srpm:	patches tarball
-	rm -f ${HOME}/rpmbuild/SRPMS/$(prog)-*.src.rpm
-	rpmbuild -bs ${HOME}/rpmbuild/SPECS/$(prog).spec
+srpm: spec patches tarball
+	rm -f ${RPMSRPM}/${prog}-*.src.rpm
+	rpmbuild -bs ${RPMSPEC}/${prog}.spec
```

Comment:
Do we still have definitions for `RPMSRPM` and `RPMSPEC`?  (Didn't we remove them from old lines 20-21?)

Reply:
They are in  the respective `rpm.mk` files. In 0.71 and later, there is just one `rpm.mk` file refactored out of the server/agent ones, but there is still a definition for the RPM* variables. IIRC...

Reply:
Ugh:  I hate it when the references appear "before" the definitions.  (But, this is Make, so, yeah, it really doesn't matter, but....)